docs/property-graph-model.adoc

```diff
@@ -0,0 +1,75 @@
+:numbered:
+
+= Property Graph Model
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+Cypher is a graph query language, but it doesn't operate on any arbitrary definition of graph; it operates on a Property Graph.
+A Property Graph may be defined in graph theoretical terms as a directed, vertex-labeled, edge-labeled multigraph with self-edges, where edges have their own identity.
```

Comment:
Also mention that we use the term "node" to denote a vertex, and "relationship" to denote an edge (since "vertex" and "edge" are used to define the property graph model)